R/format_fy.R

```diff
@@ -33,7 +33,7 @@ format_fy <- function(year) {
 # function to reverse the change back to e.g. 201617
 format_fy_reverse <- function(year) {
   if (!grepl("^\\d{4}-\\d{2}.*", year)) {
-    stop("year parameter must be a seven digit string e.g. '2016-17'")
+    stop("year parameter be in the format 2016-17")
```

Comment:
How about a blend of this and what we had before - "year parameter must a 7 digit string in a financial year format, e.g. '2017/18'"?